course5/homework2/newton/newton.cc

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <string>
+
+// Newton method of sqrt. The stack frame was 48 for Debug mode and 64 for Release mode.
+
+bool* start = nullptr;
```

Comment:
These are still global. And you don't need bool. To store pure, untyped address just use void*